products/spiders/books.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict, Any
+
+import scrapy
+from scrapy.http import Response
+
+
+RATING = {
+    "One": 1,
+    "Two": 2,
+    "Three": 3,
+    "Four": 4,
+    "Five": 5,
+}
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def parse(self, response: Response, **kwargs) -> str:
```

Comment:
use correct type annotation, It wouldn't be str

Reply:
change it, think this is correct